sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -29,24 +31,27 @@ This table shows the relationship between SDK versions and supported API version
 
 |SDK version|Supported API version of service
 |-|- |
+|5.2.0 | 3.0, 3.1, 2022-05-01 (default)
 |5.2.0-beta.4 | 3.0, 3.1, 2022-05-01 (default)
```

Comment:
do we need to remove these betas.

Reply:
Probably a good idea. Apart from the CHANGELOG, we don't make mention elsewhere of betas.